package.json

```diff
@@ -23,21 +23,26 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-profile",
   "dependencies": {
+    "@digitalbazaar/did-method-key": "^1.0.0",
+    "@digitalbazaar/ed25519-signature-2018": "^2.0.1",
+    "@digitalbazaar/ed25519-signature-2020": "^2.1.0",
+    "@digitalbazaar/ed25519-verification-key-2018": "^3.1.1",
```

Comment:
I think this list also needs `@digitalbazaar/ed25519-verification-key-2020` dep? (Since we're importing it in `lib/utils.js`)

Reply:
added here 4ab3f41